/Users/vladimirkopylov/bots/venv/bin/python3: can't open file '/Users/vladimirkopylov/bots/bots/panictab_bot/panictab_bot.py': [Errno 2] No such file or directory
